docs/python-reference/index.md

```diff
@@ -1,5 +1,7 @@
 # Python Reference
 
+![Codecov](https://codecov.io/github/prairielearn/prairielearn/graph/badge.svg?token=50XtugkgLU&flag=python)
```

Comment:
This shows up as an unknown flag:

![Codecov](https://codecov.io/github/prairielearn/prairielearn/graph/badge.svg?token=50XtugkgLU&flag=python)

Do we need to land this on master first? FWIW, I can see the flag info from the web UI:

<img width="1573" alt="Screenshot 2025-02-28 at 12 51 26" src="https://github.com/user-attachments/assets/6a7e38b1-0d9e-40a5-9842-31d33eb168cf" />

In fact I'm pretty sure this URL is publicly accessible: https://app.codecov.io/gh/PrairieLearn/PrairieLearn/flags